app/src/sharedTest/java/org/oppia/app/topic/play/TopicPlayFragmentTest.kt

```diff
@@ -42,7 +39,6 @@ import javax.inject.Singleton
 class TopicPlayFragmentTest {
```

Comment:
<img width="1220" alt="Screenshot 2019-10-26 at 12 03 17 PM" src="https://user-images.githubusercontent.com/16301028/67615375-231c2b00-f7e9-11e9-90c9-61b0d51f4980.png">
Getting these errors while testing.

Reply:
Ah, I didn't change it in TopicActivityPresenter. Now it works. LGTM.